src/test/java/org/fundacionjala/coding/jimmy/Kata2Test.java

```diff
@@ -0,0 +1,59 @@
+package org.fundacionjala.coding.jimmy;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Test class created by Jimmy Romero.
+ */
+public class Kata2Test {
+
+    private Kata2 kata2;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AIndentationCheck)